src/index.ts

```diff
@@ -58,7 +59,8 @@ export type FSWInstanceOptions = BasicOpts & {
 };
 
 export type ThrottleType = 'readdir' | 'watch' | 'add' | 'remove' | 'change';
-export type EmitArgs = [EventName, Path | Error, any?, any?, any?];
+export type EmitArgs = [Path | Error, Stats?];
```

Comment:
Is that OK for backwards compat?

Reply:
Sorry I forgot to reply to this. We do export it so it's technically a breaking change to the types. But as far as I'm aware, it's not part of our "public API". 

I couldn't find any usages of it in the wild. So I think it's really just an internal type exported for completeness